src/second_ode_solve.jl

```diff
@@ -0,0 +1,64 @@
+struct NNODE2{C,O} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+end
+NNODE2(chain;opt=Flux.ADAM(0.1)) = NNODE2(chain,opt)
+
+function DiffEqBase.solve(
+    prob::DiffEqBase.AbstractODEProblem,
+    alg::NeuralNetDiffEqAlgorithm,
+    args...;
+    dt,
+    timeseries_errors = true,
+    save_everystep=true,
+    adaptive=false,
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 100)
+
+    DiffEqBase.isinplace(prob) && error("Only out-of-place methods are allowed!")
+
+    u0, du0 = prob.u0
+    tspan = prob.tspan
+    f = prob.f
```

Comment:
```suggestion
    f = prob.f.f1
```

Reply:
it's from how it's handled internally, so 1st order ODE solvers can work with it: https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/problems/ode_problems.jl#L143-L176

Reply:
I see, sure!